package.json

```diff
@@ -26,6 +26,7 @@
     "autoprefixer": "^7.1.1",
     "autosize": "^3.0.20",
     "babel-runtime": "^6.23.0",
+    "bcrypt": "^1.0.3",
```

Comment:
We've removed `bcrypt` because it's been causing errors in our build process, so while it complains when you start up, it's been removed on purpose.

Reply:
Yeah, this was a mistake, gonna remove it on a next commit and when everything is cool, we just squash all commits into one and those changes will disappear.